test/libsolidity/smtCheckerTests/types/bytes_2.sol

```diff
@@ -7,3 +7,5 @@ contract C
 		assert(b1[1] == b2[1]);
 	}
 }
+// ----
+// Warning 6328: (119-141): CHC: Assertion violation happens here.\nCounterexample:\n\n\nTransaction trace:\nC.constructor()\nC.f(b1, b2)
```

Comment:
@leonardoalt , basically, most of the code I have showed you today was to address this introduced false positive. I could add the code here, but I think it is better to do it gradually, so I would do it in another PR.

Reply:
Yep, makes sense!